packages/html-reporter/src/testFileView.tsx

```diff
@@ -87,6 +87,10 @@ function traceBadge(test: TestCaseSummary): JSX.Element | undefined {
   return firstTraces ? <Link href={generateTraceUrl(firstTraces)} title='View trace' className='test-file-badge'>{trace()}</Link> : undefined;
 }
 
+export function emptyPathFilter(path: string[]): string[] {
```

Comment:
Ideally, we would fix this when generating the report instead of when rendering it, specifically at this line:

https://github.com/microsoft/playwright/blob/main/packages/playwright/src/reporters/html.ts#L362

Reply:
Thanks for the hint

I've fixed it where you pointed it out.